azure-pipelines.windows.yml

```diff
@@ -0,0 +1,178 @@
+name: Roaster Windows Build
+
+trigger:
+- master
+
+pool:
+  name: $(AGENT_POOL_NAME)
+
+stages:
+- stage: prep
+  jobs:
+    - job: vsbuildtools
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+        parameters:
+          stage_name: vsbuildtools
+
+    - job: cmake
+      dependsOn: vsbuildtools
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: curl
+      dependsOn: cmake
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: cuda
+      dependsOn: curl
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: cudnn
+      dependsOn: cuda
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: tensorrt
+      dependsOn: cudnn
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: intel
+      dependsOn: tensorrt
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: mklml
+      dependsOn: intel
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+- stage: build
+  dependsOn: prep
+  jobs:
+    - job: zlib
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: openssl
+      dependsOn: zlib
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: c_ares
+      dependsOn: openssl
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+        parameters:
+          stage_name: c-ares
+
+    - job: freetype_fp
```

Comment:
No idea what's fp/sp, certainly not first/second pass →_→
Even 1/2 works better...

Reply:
ok.. i just want to mean `first_pass` and `second_pass`....

Reply:
fxied.